Jenkinsfile

```diff
@@ -5,15 +5,15 @@
 def clone_test_deps() {
   dir("${WORKSPACE}") {
     sh "git clone git@github.com:xmos/test_support"
-    sh "git -C test_support checkout c820ebe67bea0596dabcdaf71a590c671385ac35"
+    sh "git -C test_support checkout remove_whitespace"
```

Comment:
I guess this needs changing?

Reply:
https://github.com/xmos/test_support/pull/50 needs to be merged before I can change it.

Reply:
done